src/modules/service/token-service.js

```diff
@@ -0,0 +1,53 @@
+const jwt = require('jsonwebtoken');
+const tokenModel = require('../../models/token');
+
+class TokenService {
+  generateTokens(payload) {
+    const accessToken = jwt.sign(payload, process.env.JWT_ACCESS_SECRET, { expiresIn: '30m' });
```

Comment:
process.env.JWT_ACCESS_SECRET
process.env.JWT_REFRESH_SECRET

this will be in config.js